server/server/data_loader/arangoload.py

```diff
@@ -149,6 +157,14 @@ def process_root_files(docs, edges, mapping, root_files, root_languages):
 
             del entry['_path']
             entry['num'] = i  # number is used for ordering, it is not otherwise meaningful
+            try:
+                entry['acronym'] = sutta_data[uid]['acronym']
+            except KeyError:
+                pass
```

Comment:
errors passing silently?

Reply:
Yes. Because not all files in root collection refer to suttas, some refer to vaggas and higher level and in that case, there is no acronym for given UID.